CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## next
 
+## 1.1.0
+
 *Bug Fixes*
 - Fix a bug causing secret generation from ejson to fail when decryption succeeded but a warning was also emitted. [#647](https://github.com/Shopify/krane/pull/647)
```

Comment:
@timothysmith0609 is this the changelog entry you mean?

Reply:
:facepalm: I don’t appear to be awake today

Reply:
:facepalm: I don’t appear to be awake today

Reply:
Haha it's OK <3